plugins/wmm_pi/src/WMM_SubLibrary.c

```diff
@@ -1790,21 +1790,21 @@ WMMtype_LegendreFunction *WMM_AllocateLegendreFunctionMemory(int NumTerms)
 	if (!LegendreFunction) {
 		WMM_Error(1);
 		//printf("error allocating in WMM_AllocateLegendreFunctionMemory\n");
-		return FALSE;
+		return NULL;
 					}
 	LegendreFunction->Pcup = (double *) 	malloc	( (NumTerms +1) * sizeof ( double ) );
 	if (LegendreFunction->Pcup == 0)
 	{
 		WMM_Error(1);
 		//printf("error allocating in WMM_AllocateLegendreFunctionMemory\n");
-		return FALSE;
+        return NULL;

```

Comment:
Oh, might be the case, but my expectation would be to not only change the line to spaces but at least the whole function.

Sorry to bug you about it if another rule was agreed on.